packages/config/config/devices/0x0371/zwa039.json

```diff
@@ -18,8 +18,8 @@
 		}
 	],
 	"firmwareVersion": {
-		"min": "2.0",
-		"max": "10.255"
```

Comment:
Wasn't firmware 11.x+ our special case - v1 revision with different firmware?

Reply:
Yes, I just missed that because the 0-1.255 and 11-255.255 have a different model number (ZWA009). Lint should check for overlaps like this, shouldn't it?